### PR TITLE
Fun 125

### PR DESCRIPTION
**Pull request by adrielstar**
Check if the response house objects are in the searched "buurt" polygon.
if not what's will happen ?

### CODE REVIEW

File: application/views/scripts/result/index.phtml [hunk 1]
```diff
@@ -133,72 +185,54 @@
...
-                    infoPopUp.setContent(detailsPopUp);
-                    infoPopUp.open(map, houseMarker);
-                    houseMarker.setIcon(icon2);
-                });
+$point1 = geoPHP::load("POINT($latPoint $lngPoint)","wkt");
 
-                google.maps.event.addListener(houseMarker, 'mouseout', function () {
-                    infoPopUp.close();
-                    houseMarker.setIcon(icon);
+$polygone = geoPHP::load("$polygonBuurt","wkt");
```
Comment (isaac-youwe) [inline application/views/scripts/result/index.phtml]: $polygon

File: application/views/scripts/result/index.phtml [hunk 2]
```diff
@@ -82,7 +82,59 @@
...
+                        return;
+                    }
+
+                    map.panTo(lastValidCenter);
+                });
+            }
+
+            google.maps.event.addDomListener(window, 'load', initialize);
+
+            function addInfoWindow(houseMarker, details) {
```
Comment (isaac-youwe) [inline application/views/scripts/result/index.phtml]: @adrielstar I miss the function documentation.

File: application/views/scripts/result/index.phtml [hunk 3]
```diff
@@ -133,72 +185,54 @@
...
-                    houseMarker.setIcon(icon2);
-                });
+$point1 = geoPHP::load("POINT($latPoint $lngPoint)","wkt");
 
-                google.maps.event.addListener(houseMarker, 'mouseout', function () {
-                    infoPopUp.close();
-                    houseMarker.setIcon(icon);
+$polygone = geoPHP::load("$polygonBuurt","wkt");
+var_dump($polygone->contains($point1));
+echo "Is all hous in polygon? " . ($polygone->contains($point1) ? "YES" : "NO");
```
Comment (flexi758) [inline application/views/scripts/result/index.phtml]: the echo is not correct english
dont know if this is important?
@adrielstar @isaac-youwe

Comment (isaac-youwe) [inline application/views/scripts/result/index.phtml]: @flexi758 I think this Pull request is just a work in progress. I see var_dump and a lot of commented code so I don't think this is ready to merge.

**Comment by isaac-youwe** [comment]
@flexi758 Can you fetch this branch and test it on your machine? I'm running the script in chunks so it will take a while to finish meaning I can not test this atm

**Comment by isaac-youwe** [comment]
can you send some search terms so that we can test this? @adrielstar

File: application/views/scripts/result/index.phtml (hunk 3 above)
Comment (adrielstar) [inline application/views/scripts/result/index.phtml]: like i said this is still in work progress. i have test it on Appingedam/Appingedam-Oost of West of Centrum

File: application/views/scripts/result/index.phtml [hunk 4]
```diff
@@ -82,7 +82,59 @@
...
+                        houseMarker.open = true;
+                    }
+                    else {
+                        infoWindow.close(map, houseMarker);
+                        houseMarker.open = false;
+                    }
+                });
 
+            }
+            function addPopupWindow(houseMarker, detailsPopUp) {
```
Comment (adrielstar) [inline application/views/scripts/result/index.phtml]: @isaac-youwe  yes it is possible. I just change it so i can get a good stricture.

**Comment by isaac-youwe** [comment]
FUN-125 is merged please update you're branch.

**Comment by isaac-youwe** [comment]
@adrielstar how about the list? you are only filtering the map. the list should also be filtered. 
image

**Pull request merged**